src/services/helpers-service.ts

```diff
@@ -70,7 +71,7 @@ export class HelpersService {
 			return array;
 		}
 
-		if ("TextEncoder" in window) {
+		if (this.isBrowser() && "TextEncoder" in window) {
```

Comment:
would be more readable with ("TextEncoder" in window)

Reply:
would also throw an exception in nodejs :D